core/src/main/scala/caliban/GraphQLInterpreter.scala

```diff
@@ -23,14 +23,19 @@ trait GraphQLInterpreter[-R, +E] { self =>
    * Parses, validates and finally runs the provided request against this interpreter.
    * @param request a GraphQL request
    * @param skipValidation skips the validation step if true
+   * @param introspectionEnabled returns an error for introspection queries when false
```

Comment:
For naming consistency with `skipValidation`, how about naming it `enableIntrospection`?

Reply:
Done!